6_drivers/src/nldas_feather_utils.R

```diff
@@ -99,7 +101,8 @@ cubes_to_cell_file <- function(filename, ..., nc_files = NULL){
   cell_out <- data.frame(data = rep(NA_real_, length(cell_time_indices))) %>% setNames(cell_var)
 
   if (is.null(nc_files)){
-    nc_files <- c(...)
+    nc.ind_files <- c(...)
+    nc_files <- file.path(nc_dir, tools::file_path_sans_ext(basename(nc.ind_files)))
```

Comment:
you could still use `scipiper::as_data_file` in place of `tools::file_path_sans_ext` here, even though you're also swapping out the dir path - might make it slightly clearer what you're doing, or not...